### DISCUSSION TITLE
Extracting the SM values of the WET couplings at a given scale.

### DISCUSSION THREAD

**Discussion by Luc-Darme** [flav-io/flavio#286] [Q&A] [answered]
Dear authors,

I was wondering if there was a simple way of extracting from Flavio the SM values of the WET couplings for a given sector and at a given scale?

For the particular case of b->s physics, I could hack through wctot_dict from bdecays/wilsoncoefficients.py but it does not seem a very elegant solution (and the couplings are not in the Flavio basis so I can not directly pass them to Wilson to run them at the scale I need).

All the best
Luc

**Comment by DavidMStraub** [comment, marked as answer]
Hi,

not really, and this is intentional (at least when it comes to the original implementation) because "SM values of the WET couplings" is an ambiguous concept. Renormalization scheme dependence, long-distance QED effects, etc. etc. This is why WCXF only considers BSM contributions, which are always short-distance effects.
